### PR TITLE
Create script-StringContains.yml

### PR DESCRIPTION
**Pull request by meirwah**
condition script

### CODE REVIEW

File: Scripts/script-StringContains.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  var index = str.indexOf(args.substring);
+  if (index > -1) {
+      return 'yes';
+  }
+  return 'no';
+type: javascript
+tags:
+- Condition
+- Utility
+comment: "Contains reports whether substring is within string.\nreturns :\nyes - string contains
```
Comment (slavikm) [inline Scripts/script-StringContains.yml]: No `\n`, please

**Comment by dorsha** [comment]
@slavikm - I removed the \n, please check and merge.

**Comment by dorsha** [comment]
Any idea what is the problem?
image

**Comment by dorsha** [comment]
Fixed.

**Pull request merged**